styles/styles.css

```diff
@@ -1,124 +1,112 @@
-h1 {
-    color: #993f3f; text-align: center;
-}
-#blockquote {
-    color: #3b3a3a;
-}
-h2 {
-   color: #1f4796;
-}
-h3 {
-    color: #4d8d4d;
-    margin-bottom: 30px;
-}
-h4 {
-    margin-bottom: 30px;
-}
-h5 {
-    margin-bottom: 30px;
-}
-.img {
-    max-width: 350px;
-    min-height: 450px;
-}
-table, td, th {
-     border: 1px solid black;
- }
-table {
-    border-collapse: collapse;
-    width: 80%;
-}
-th {
-    height: 50px;
-    text-align: left;
-}
-tr:hover {
-    background-color: #deba97;
-}
-a:hover {
-    color: #3b3a3a;
-}
-input[type="text"]
-{
-    color: #e79898;
-}
-ol::before {
-    content: " ! The first step towards the development !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
-}
-ol::after {
-    content: " ! The first step towards the development has been taken !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
-}
-.input {
-    min-inline-size: auto;
-    /*height: 100px;*/
+@import url('./fonts');
+
+body {
+    font-family: 'Roboto', sans-serif;
 }
-.button {
-    min-width: 70px;
+
+*{
+    margin: 0;
+    box-sizing: border-box;
 }
-.form_fieldset {
-    margin-bottom: 30px;
+
+/*Lesson 5*/
+/*WHAT DO WE OFFER*/
+
+.title {
+    text-align: center;
+    color: #383838;
 }
-/*Lesson 4 */
-.advantage {
-    max-width: 1480px;
+
+.cont {
+    max-width: 1440px;
+    padding: 100px 16px 0;
     margin: 0 auto;
 }
-.advantage__title {
-    text-align: center;
-    font-style: normal;
-    font-weight: 500;
+
+.cont_title {
     font-size: 40px;
-    line-height: 90%;
-    color: #333333;
-    margin-bottom: 50px;
+    font-weight: 500;
+    text-align: center;
+    margin: 120px auto 20px;
 }
-.advantage__title__line {
+
+.cont_line {
     border: none;
     background: #F5BD41;
     overflow: visible;
     width: 100px;
     height: 4px;
-    margin: 0 700px 50px;
-}
-.advantage__list {
-    list-style: none;
-}
-.advantage__item {
-    display: inline-block;
-    max-width: 300px;
-    margin-right: 200px;
-    vertical-align: top;
+    margin: 70px auto 100px;
 }
-.advantage__item:last-child {
-    margin-right: 0;
+
+.row {
+    display: flex;
+    gap: 16px;
+    margin: 20px 104px 115px;
 }
-.advantage__desc {
-    font-weight: 400;
-    font-size: 24px;
-    line-height: 100%;
-    text-align: left;
-    margin: 50px 0;
-    color: #333;
+
+.col {
+    width: 25%;
 }
-.advantage__block {
-    height: 180px;
-    width: 200px;
-    margin: 0 auto 32px;
+
+.cont_row {
+    margin-bottom: 120px;
 }
-.advantage__icon {
 
+.cont_col {
+    text-align: center;
 }
-img {
+
+.cont_icon {
     max-width: 100px;
     min-height: 100px;
     width: auto;
-}
\ No newline at end of file
+}
+
+.cont_title_icon {
+    font-size: 24px;
+    line-height: 28px;
+    margin: 65px auto 10px;
+    font-weight: 400;
+    text-align: center;
+    color: #383838;
+}
+
+/*FOOTER*/
+
+.flex-cont-footer {
+    background: #10367C;
+}
+
+.cont_footer_row {
+    display: flex;
```

Comment:
you need to align content like in design. So if  you notice, all content is align by side
margin:0 auto  should help you